test/fixtures/full/variables.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "dataset_id" {
+  description = "Unique ID for the dataset being provisioned"
+}
+
+variable "dataset_name" {
+  description = "Friendly name for the dataset being provisioned"
+}
+
+variable "description" {
+  description = "Dataset description"
+}
+
+variable "expiration" {
+  description = "TTL of tables using the dataset in MS"
+}
+
+variable "project_id" {
+  description = "Project wheree the dataset and table are created"
+}
+
+variable "table_id" {
+  description = "Unique ID for the table being provisioned"
+}
+
+variable "region" {
```

Comment:
Blocker: see the earlier comment regarding regions vs zones.

Reply:
Will be changed in the next release